miso-web/src/main/webapp/scripts/list.js

```diff
@@ -298,6 +298,29 @@ ListUtils = (function($) {
           updateSelectedLabel(state);
         }
       });
+      staticActions.push({
+        "name": "🗹",
+        "title": "Select all on all pages",
+        "handler": function() {
+          if (jqTable.fnSettings().fnRecordsDisplay() > 10000) {
+            Utils.showOkDialog('Select all', [ 'Too many items selected.', 'Please use stricter filtering.' ]);
```

Comment:
Could we communicate the 10 000 item limit to users in the error message?

Reply:
Done.